apps/cyberstorm-storybook/.storybook/preview.js

```diff
@@ -0,0 +1,26 @@
+import { LinkingProvider } from "@thunderstore/components";
+import { LinkLibrary } from "../LinkLibrary";
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  backgrounds: {
+    default: "thunderstore",
+    values: [
+      { name: "thunderstore", value: "#1E1E1E" },
```

Comment:
At least on my screen this makes many of the button styles difficult to see. Should we consider some more neutral background color?

Reply:
This color is the one Figma uses. Feel free to suggest any others.

Reply:
I think @gkurck might be a more suitable suggester.

Reply:
You could use the `surface-1`, the same as the website. If that's still an issue visually, we can question the feasibility of currently assigned colors.

Reply:
It's slightly better for the default button. `button__defaultDark` currently uses that same variable, making it invisible, but I guess it's up to developer to not use such combination. Maybe that button can be rendered with some wrapped containing a different bg color in Storybook?

Reply:
now uses the surface-1